scripts/ci-entrypoint.sh

```diff
@@ -58,12 +58,15 @@ setup() {
         echo "Will use the ${IMAGE_REGISTRY}/${CCM_IMAGE_NAME}:${IMAGE_TAG_CCM} cloud-controller-manager image for external cloud-provider-cluster"
         echo "Will use the ${IMAGE_REGISTRY}/${CNM_IMAGE_NAME}:${IMAGE_TAG_CNM} cloud-node-manager image for external cloud-provider-azure cluster"
 
-        CCM_IMG_ARGS=(--set cloudControllerManager.imageRepository="${IMAGE_REGISTRY}"
-        --set cloudNodeManager.imageRepository="${IMAGE_REGISTRY}"
-        --set cloudControllerManager.imageName="${CCM_IMAGE_NAME}"
-        --set cloudNodeManager.imageName="${CNM_IMAGE_NAME}"
-        --set-string cloudControllerManager.imageTag="${IMAGE_TAG_CCM}"
-        --set-string cloudNodeManager.imageTag="${IMAGE_TAG_CNM}")
+        if [[ -n "${LOAD_CLOUD_CONFIG_FROM_SECRET:-}" ]]; then
```

Comment:
Do we still want to use `E2E_ARGS="-kubetest.use-ci-artifacts"` as the variable configuring the CI artifacts?

Reply:
`should_build_ccm` already encompasses `E2E_ARGS="-kubetest.use-ci-artifacts"` and other cases where we want to use a CI version of CCM in the context of ci-entrypoint.sh

https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/hack/util.sh#L44